tensorflow/python/types/core.py

```diff
@@ -58,3 +62,7 @@ class Value(Tensor):
 
   def numpy(self):
     pass
+
+
+TensorLike = Union[Tensor, int, float, bool, str, tuple, list, np.ndarray]
+tf_export("TensorLike").export_constant(__name__, "TensorLike")
```

Comment:
Let's use "types.experimental.TensorLike" for now. We also need to update the goldens for api_test. Lastly, we'll need to document the symbol.

Reply:
I used "experimental.TensorLike" instead to allow the goldens to be updated.